src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -466,7 +471,17 @@ private static void addConversionConfigs(ConfigDef configDef) {
         group,
         ++order,
         Width.SHORT,
-        "Write method");
+        "Write method"
+    ).define(
+        RETRY_ON_CONFLICT_CONFIG,
+        Type.INT,
+        0,
+        Importance.LOW,
+        RETRY_ON_CONFLICT_DOC,
+        group,
+        ++order,
+        Width.SHORT,
+        "Retry on conflict");
```

Comment:
I think this belongs in the connector configs group

Reply:
Makes sense. I'll change it. Thanks